pybamm/models/submodels/equivalent_circuit_elements/diffusion_element.py

```diff
@@ -0,0 +1,110 @@
+import pybamm
+
+
+class NoDiffusion(pybamm.BaseSubModel):
+    """
+    Without Diffusion element for
+    equivalent circuits.
+
+    Parameters
+    ----------
+    param : parameter class
+        The parameters to use for this submodel
+    options : dict, optional
+        A dictionary of options to be passed to the model.
+    """
+
+    def __init__(self, param, options=None):
+        super().__init__(param)
+        pybamm.citations.register("Fan2022")
```

Comment:
The paper mentions ECM with diffusion, and it is cited in the `NoDiffusion` class. I would expect this citation to be used in the `DiffusionElement` class. Should it be in both?

Reply:
We fixed it now!